src/System/Led.h

```diff
@@ -55,7 +55,7 @@ class Led {
         void pulse(uint16_t interval);
 
     private:
-        enum LED_STATE {OFF, ON, PULSE, FLASH_ON, FLASH_OFF, FLASH_REPEAT};
```

Comment:
Changed this enum as it was causing conflicts with constants that have since been removed. Better to have a more descriptive enum anyway.